tests/OAuth2ImplicitGrantTypeTest.php

```diff
@@ -30,7 +30,7 @@ public function testGrantAccessTokenWithGrantImplicit()
                 'state' => '42',
         )));
 
-        $this->assertRegExp('/^http:\/\/www.example.com\/\?foo=bar#state=42&access_token=[^"]+&expires_in=3600&token_type=bearer$/', $response->headers->get('Location'));
+        $this->assertMatchesRegularExpression('/^http:\/\/www.example.com\/\?foo=bar#state=42&access_token=[^"]+&expires_in=3600&token_type=bearer$/', $response->headers->get('Location'));
```

Comment:
does `assertMatchesRegularExpression` work in all phpunit versions supported by the constraint (that you did not update) ?